src/memory/AmdFlash.cc

```diff
@@ -728,7 +728,7 @@ static constexpr std::initializer_list<enum_string<AmdFlash::State>> stateInfo =
 	{ "IDENT",  AmdFlash::State::IDENT },
 	{ "CFI",    AmdFlash::State::CFI },
 	{ "STATUS", AmdFlash::State::STATUS },
-	{ "PRGERR", AmdFlash::State::PRGERR }
+	{ "ERROR",  AmdFlash::State::ERROR }
```

Comment:
I fully agree that bw-compat with old savestates is not needed for this (very unlikely this state occurs in a savestate).

So just FYI, if you do want this for some other thing in the future. It's fine to have multiple strings in this list that map to the same enum-value. So not removing the old line is all you need to do (and add a comment that it's for bw-compat).

Reply:
Oh nice!